docs/development/manual_tests.md

```diff
@@ -738,3 +738,128 @@ nextflow run main.nf -profile test,docker --outdir ./results -w work/ -resume --
 ## NOTE: PG tags are repeated for each chromosome in the reference, times each library! Maybe there's some flag missing from samtools MERGE runs?
 nextflow run main.nf -profile test_multiref,docker --outdir ./results -w work/ -resume --genotyping_source 'raw' -ansi-log false -dump-channels
 ```
+
+# GENOTYPING
+
+These tests were ran before library merging was implemented.
+
+## GATK UG
+
+```bash
+## Gatk UG on raw reads
+## Expect: One VCF + .tbi index per sample/reference combination. Also 1 bcftools_stats file per VCF. Additional IR/ subdirectory with 1 bam and 1 bai per sample/reference combination.
+nextflow run main.nf -profile test,docker --outdir ./results -w work/ -resume --run_genotyping --genotyping_tool 'ug' --genotyping_source 'raw' --genotyping_gatk_ug_keeprealignbam -ansi-log false -dump-channels
```

Comment:
are -dumpl-channels kept in the manual test?

Reply:
I leave it in to ensure that I remembered to remove all `.dump()` calls